[2024-04-12T18:37:41.766+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:37:38.757551+00:00 [queued]>
[2024-04-12T18:37:41.787+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:37:38.757551+00:00 [queued]>
[2024-04-12T18:37:41.789+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-04-12T18:37:41.825+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_qdrant> on 2024-04-12 18:37:38.757551+00:00
[2024-04-12T18:37:41.865+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '12345678', 'insert_qdrant', 'manual__2024-04-12T18:37:38.757551+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpx6_rzo_m']
[2024-04-12T18:37:41.868+0000] {standard_task_runner.py:88} INFO - Job 20: Subtask insert_qdrant
[2024-04-12T18:37:41.854+0000] {standard_task_runner.py:60} INFO - Started process 88 to run task
[2024-04-12T18:37:41.961+0000] {task_command.py:423} INFO - Running <TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:37:38.757551+00:00 [running]> on host 5feba290cccc
[2024-04-12T18:37:42.049+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='12345678' AIRFLOW_CTX_DAG_ID='12345678' AIRFLOW_CTX_TASK_ID='insert_qdrant' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T18:37:38.757551+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T18:37:38.757551+00:00'
[2024-04-12T18:37:42.067+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'message': 'Data inserted to Vector DB successfully'}
[2024-04-12T18:37:42.107+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=12345678, task_id=insert_qdrant, execution_date=20240412T183738, start_date=20240412T183741, end_date=20240412T183742
[2024-04-12T18:37:42.143+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-12T18:37:42.162+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
